infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/DefaultSiteConfig.java

```diff
@@ -31,7 +31,7 @@ private DefaultSiteConfig() {
     private static String primaryCorfuPort = "9000";
 
     @Getter
-    private static String primaryLogReplicationPort = "9010";
+    private static List<String> primaryLogReplicationPorts = Arrays.asList("9010", "9011", "9012");
```

Comment:
This file is no longer referenced in the code, can you rebase and please modify in DefaultClusterConfig.java instead.